charts/opentelemetry-demo/templates/_pod.tpl

```diff
@@ -106,7 +112,7 @@ Exclude email service and treat differently because the addr. for the email serv
 
 {{- if eq .name "product-catalog-service" }}
 - name: FEATURE_FLAG_GRPC_SERVICE_ADDR
-  value: {{ (printf "%s-featureflag-service:50031" $prefix ) }}
+  value: {{ (printf "%s-featureflag-service:50053" $prefix ) }}
```

Comment:
Is this related to the shipping service? If not, let's put it in its own PR.

Reply:
I thought this was unrelated and removed it, but we actually need this so that the checkout service will complete it's flow to shipping and quote services.